src/ai/backend/manager/reporters/hub.py

```diff
@@ -0,0 +1,67 @@
+import asyncio
+from dataclasses import dataclass
+
+from ai.backend.manager.reporters.base import (
+    AbstractReporter,
+    FinishedActionMessage,
+    StartedActionMessage,
+)
+
+
+@dataclass
+class ReporterHubArgs:
+    reporters: dict[str, list[AbstractReporter]]  # Key: action type, Value: reporter instance
+    concurrency_limit: int = 5
+
+
+class ReporterHub:
```

Comment:
ReporterHub is not `AbstractReporter`, but use `report_started`, `report_ended`. 
Is this inteneded?

Reply:
That was the intention, but it looks like it's okay to inherit it together.